plan/conflict.go

```diff
@@ -59,7 +59,7 @@ func (s PerResource) ResolveUpdate(current *endpoint.Endpoint, candidates []*end
 			return ep
 		}
 	}
-	return s.ResolveCreate(candidates)
```

Comment:
What is the reason for changing it?

Reply:
The logic of ResolveCreate hopes that the exact matching method is different. After all, the logical update logic of multiple records is different from the processing of a single record. For example, a single record is mostly in the update mode, and multiple records are created first. In the way of delete, is there a better way for this piece?

Reply:
This logic thinks about it, the weight setting this scenario is a multi-cluster solution, the single cluster should maintain a 1:1 correspondence, I am modifying the PR, then apply for review, hard